src/main/resources/External_Resources.json

```diff
@@ -0,0 +1,1030 @@
+[
```

Comment:
Why is there a JSON file? Didn't I just see the same content in a CSV file?

Reply:
Yes, initially I had the resource information in a Google Sheet and was building the parser based on CSV, but I realized JSON was a better way to go.  The application supports both right now, but I may move it to entirely JSON.

Reply:
Created issue #6